### ISSUE TITLE
Clarify URL for "The Lux Programming Language"

### ISSUE THREAD

**Issue by phlummox** [LuxLang/lux#53]
The link given in the [README][readme-book] for the  "The Lux Programming Language" book is <https://www.gitbook.com/book/luxlang/the-lux-programming-language/details> which redirects to <https://legacy.gitbook.com/book/luxlang/the-lux-programming-language/details>. That takes me to a page which shows nothing but "Please log in to preview this content", and apparently logging in requires me to provide a phone number ... uh, no. Even if the number *is* thrown away after use.

I see there's also, apparently, a version of the book at <https://luxlang.gitbooks.io/the-lux-programming-language/>. Do these two URLs provide exactly the same content? If so, it might be worth mentioning this latter URL in the README. 

[readme-book]: https://github.com/LuxLang/lux/blob/cf9e65352cb477982754c98fafe0a3a98a42670d/README.md#where-do-i-learn-lux

**Comment by eduardoejp** [comment]
Hello, @phlummox !

I apologize for taking so long to answer. I just realized I wasn't "watching" this repository, so I didn't get any notification from GitHub when you created this issue.
I starting "watching" the repo, so that mistake won't happen again.

Anyhow, I just tried [the link that you gave](https://www.gitbook.com/book/luxlang/the-lux-programming-language/details), and [it did redirect me](https://legacy.gitbook.com/book/luxlang/the-lux-programming-language/details), as you mentioned. But the web page I see has a green button to the right that says "Read", which takes me to the book without needing to log-in.

Does the "Read" button not display for you?

If so, [the third link you provided](https://luxlang.gitbooks.io/the-lux-programming-language/) does indeed take you to the book, with the same content.
